spec/services/github_spec.rb

```diff
@@ -2,39 +2,36 @@
 
 describe Github do
   context "when given a repo" do
+    let(:args) { {org_repo: "CodeMontageHQ/codemontage",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space inside { missing.